Update user docu especially with regard to speed monitoring (#303)
PilzDE/pilz_robots: prbt_hardware_support/README.md

#### Table of Contents
- [Introduction](#introduction)
- [Supported DIN EN ISO 10218-1 safety features](#supported-din-en-iso-10218-1-safety-features)
  - [Needed/supported hardware](#neededsupported-hardware) 
  - [Definitions](#definitions)
  - [Safe stop 1](#safe-stop-1-ss1)
  - [Brake test](#brake-test)
  - [Operation modes](#operation-modes)
  - [Speed monitoring](#speed-monitoring)
- [Overview system components](#overview-system-components)

# Introduction
The `prbt_hardware_support` package provides support for the Pilz hardware

> PNOZmulti and PSS 4000. A number of safety feature are provided which are

Reviewer: ```suggestion
PNOZmulti and PSS 4000. A number of safety features are provided which are
```

essential for a DIN EN ISO 10218-1 certifiable robot system (for more
information see section about 
[supported DIN EN ISO 10218-1 safety features](#supported-din-en-iso-10218-1-safety-features)).  
  
Due to the fact, that the communication between the Pilz safety controllers 
and ROS is based on the Modbus communication protocol, the package also 
contains C++ files providing ROS support for the Modbus communication protocol.
  
In the [system overview section](#overview-system-components),
you can find a component diagram showing the overall architecture of our

> system. The component diagram shows all our nodes and the connection 

Reviewer: ```suggestion
system. The component diagram shows all our nodes and the connections 
```

between them.

**Please note:**  

> The launch files included in the `prbt_hardware_support` package, don't need to

Reviewer: ```suggestion
The launch files included in the `prbt_hardware_support` package don't need to
```

> be called directly by the user. They already are included in our overall

Reviewer: ```suggestion
be called directly by the user. They already are included in our top-level
```

launch file (for more information on how-to run the PRBT robot with ROS see the 
[pilz_robots-README](https://github.com/PilzDE/pilz_robots#running-on-the-real-robot).

# Supported DIN EN ISO 10218-1 safety features
The prbt_hardware_support package provides support for a number of safety
features which are essential for a DIN EN ISO 10218-1 certifiable robot system.
  
**Please note:**  
The DIN EN ISO 10218-1 support is WORK IN PROGRESS!!!  
 
## Needed/supported hardware
In order for the safety features to work, one needs certain hardware

> components supporting certain safety feature.

Reviewer: suggestion:

> In order for the safety features to work, one needs special hardware
components supporting special safety features.

Author: Done.

Currently, we test all our features against the following 
hardware setup:
- Robot: Manipulator module PRBT
- Safety controller: PSS 4000 (with special PSS 4000 program)
- Electronic selector switch (for operation mode): PITmode
- Enabling switch: PITenable
- Pushbutton unit (with emergency stop): PITgatebox

**Please note:**  
Not all safety features in this section are supported by both PILZ safety 
controllers! Currently, the PNOZmulti does not support the brake test feature 
(for more information about the brake test feature see the 

> [brake test section](#brake-test)).

Reviewer: @dbakovic Do you approve the description here?

Author: @dbakovic was asked. He approved everything so far. The minor requested change was made. -> Done.
